pkg/services/sqlstore/stats.go

```diff
@@ -133,6 +135,22 @@ func GetAdminStats(query *m.GetAdminStatsQuery) error {
 	return err
 }
 
+func GetSystemUserCountStatsCtx(ctx context.Context, query *m.GetSystemUserCountStatsQuery) error {
+	return withDbSession(ctx, func(sess *DBSession) error {
+
+		var rawSql = `SELECT COUNT(id) AS Count FROM ` + dialect.Quote("user")
+		var stats m.SystemUserCountStats
+		_, err := sess.SQL(rawSql).Get(&stats)
+		if err != nil {
+			return err
+		}
+
+		query.Result = &stats
+
+		return err
+	})
+}
+
```

Comment:
Is there a reason for adding this besides a demonstration? It seems like if we want to have `ensureAdminUser` use a transaction it would need to also wrap the dispatch of the create user command.

Reply:
Nope. Should have mentioned that in the PR

Reply:
Probably still a good idea to wrap the create user command in the transaction to make it a good demonstration